src/main/java/frc5124/robot2020/commands/shooter/SetShootRPM.java

```diff
@@ -0,0 +1,51 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc5124.robot2020.commands.shooter;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc5124.robot2020.RobotMap;
+import frc5124.robot2020.subsystems.Shooter;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+
+public class SetShootRPM extends CommandBase {
+  private Shooter shooter;
+  private double targetVelocity;
+  
+  /**
+   * Creates a new setShootVelocity.
+   */
+  public SetShootRPM (Shooter subsystem, double targetVelocity) {
```

Comment:
don't pass target velocity from the button binding. Subsystem should know which RPM constant to set as target